src/main/java/com/google/sps/servlets/GetImageServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.googl.sps.servlets;
```

Comment:
We need to add unit tests for new functionality in the same PR that we add the functionality, not in a future one. That's one time that splitting a PR is not the right way to go.